packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -9,20 +9,17 @@ import { loadingIndicator } from 'src/components/LoadingIndicator';
 import defaultClasses from './category.css';
 
 const Category = props => {
-    const {
-        id,
-        currentPage,
-        pageSize,
-        prevPageTotal,
-        setCurrentPage,
-        setPrevPageTotal
-    } = props;
+    const { id, pageSize } = props;
+
+    const [paginationValues, paginationApi] = usePagination();
+    const { currentPage, totalPages } = paginationValues;
+    const { setCurrentPage, setTotalPages } = paginationApi;
 
     const pageControl = {
-        currentPage: currentPage,
+        currentPage,
         setPage: setCurrentPage,
-        updateTotalPages: setPrevPageTotal,
-        totalPages: prevPageTotal
+        updateTotalPages: setTotalPages,
+        totalPages
```

Comment:
Similar to how I shared the Toast state could we also create a `Pagination` context and use it to share these props rather than passing them down? We already have the hooks so it feels like a pretty simple win. Not sure if you already talked about that with @jimbo though.

Reply:
In talking with @jimbo, we decided to leave the prop drilling in place, and when we re-factor `Paginiation`, we'll come back and fix it here.